internal/eventstore/models/aggregate.go

```diff
@@ -0,0 +1,87 @@
+package models
+
+import (
+	"time"
+
+	"github.com/caos/zitadel/internal/errors"
+)
+
+type AggregateType string
+
+func (at AggregateType) String() string {
+	return string(at)
+}
+
+type Aggregates []*Aggregate
+
+type Aggregate struct {
+	id             string
+	typ            AggregateType
+	latestSequence uint64
+	version        Version
+
+	editorService string
+	editorUser    string
+	editorOrg     string
```

Comment:
do we need editor org? i think its the same as resource owner

Reply:
I think it's nice if we have the date from beginning and it's easy to collect them